src/query/helper.rs

```diff
@@ -677,6 +677,39 @@ pub trait QueryFilter: Sized {
     }
 }
 
+/// Select specific column for partial model queries
+pub trait SelectColumns {
```

Comment:
I was thinking, since now we are in the same crate. We do not need this now?

Reply:
I want to limit the `PartialModelTrait` can only modify columns. If directly giving a `QuerySelect`, the implement might has unexpected effect like setting the `offset` and `limit`. It might make the behave not the same as the semantics, because the function is `into_partial_model` not `into_partial_model_with_other_query_select_effect`

Reply:
I see your concern, it's a valid point. Please move this definition to `src/query/traits.rs` then.